src/Symfony/Component/HttpKernel/Log/DebugLoggerConfigurator.php

```diff
@@ -30,7 +30,16 @@ public function __construct(callable $processor, ?bool $enable = null)
     public function pushDebugLogger(Logger $logger): void
     {
         if ($this->processor) {
+            $processors = $logger->getProcessors();
+            while ([] !== $logger->getProcessors()) {
+                $logger->popProcessor();
+            }
```

Comment:
Really not sure about this. Adding tag priority like suggested here https://github.com/symfony/monolog-bundle/pull/455 might help